src/screens/Notification.js

```diff
@@ -0,0 +1,314 @@
+        </Property>
+
+        <Property name="notification.button.color">
+          <Description>The color of the close button icon.</Description>
+          <GenericColor />
+          <PropertyValue>
+            <Example defaultValue>
+              {`
+                    color: {
+                        light: 'black',
+                        dark: 'white',
+                    },`}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.unknown.icon">
+          <Description>
+            The default icon for an unknown status Notification.
+          </Description>
+          <PropertyValue type="node">
+            <Example defaultValue>{`<StatusUnknownSmall />`}</Example>
+            <Example>{`<Icon />`}</Example>
+          </PropertyValue>
+          <PropertyValue type="SVG">
+            <Example>{`<svg>...</svg>`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.unknown.color">
+          <Description>The color of the close button icon.</Description>
+          <GenericColor />
+          <PropertyValue>
+            <Example defaultValue>
+              {`
+                    "status-unknown"
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.normal.icon">
+          <Description>
+            The default icon for a normal status Notification.
+          </Description>
+          <PropertyValue type="node">
+            <Example defaultValue>{`<StatusGoodSmall />`}</Example>
+            <Example>{`<Icon />`}</Example>
+          </PropertyValue>
+          <PropertyValue type="SVG">
+            <Example>{`<svg>...</svg>`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.normal.color">
+          <Description>The color of the close button icon.</Description>
+          <GenericColor />
+          <PropertyValue>
+            <Example defaultValue>
+              {`
+                    "status-ok"
+                `}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.warning.icon">
+          <Description>
+            The default icon for a warning status Notification.
+          </Description>
+          <PropertyValue type="node">
+            <Example defaultValue>{`<StatusWarningSmall />`}</Example>
+            <Example>{`<Icon />`}</Example>
+          </PropertyValue>
+          <PropertyValue type="SVG">
+            <Example>{`<svg>...</svg>`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="notification.warning.color">
+          <Description>The color of the close button icon.</Description>
+          <GenericColor />
+          <PropertyValue>
+            <Example defaultValue>
+              {`
+                    "status-warning"
+                `}
+            </Example>
```

Comment:
Same alignment comment as above. Show "status-warning" as the string example instead of "brand". "status-warning" doesn't need to be wrapped in {``}. Only examples that take up multiple lines or have special characters need to be wrapped with {``}. Also PropertyValue should have a type prop.